feat: add instructions to import in replit (#48374)
freeCodeCamp/freeCodeCamp: client/src/pages/learn/back-end-development-and-apis/basic-node-and-express/index.md

Express, while not included with Node.js, is another module often used with it. Express runs between the server created by Node.js and the frontend pages of a web application. Express also handles an application's routing. Routing directs users to the correct page based on their interaction with the application. While there are alternatives to using Express, its simplicity makes it a good place to begin when learning the interaction between a backend powered by Node.js and the frontend.

Working on these challenges involves writing code on Replit in our starter project.

> - Start by Importing the project on Replit.

Reviewer: The word "importing" should be lowercase:

```suggestion
- Start by importing the project on Replit.
```

Author: I totally missed on that haha :P

Express, while not included with Node.js, is another module often used with it. Express runs between the server created by Node.js and the frontend pages of a web application. Express also handles an application's routing. Routing directs users to the correct page based on their interaction with the application. While there are alternatives to using Express, its simplicity makes it a good place to begin when learning the interaction between a backend powered by Node.js and the frontend.

> Working on these challenges will involve you writing your code on Replit on our starter project. After importing the project on Replit, you will see a <code>.replit</code> window. Select <code>Use run command</code> and click the <code>Done</code> button. After completing each challenge you can copy your public Replit URL (to the homepage of your app) into the challenge screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.

Reviewer 2: ```suggestion
Working on these challenges involves writing code on Replit in our starter project. Start by Importing the project on Replit. Next, you will see a <code>.replit</code> window. Select <code>Use run command</code> and click the <code>Done</code> button. Complete each challenge and copy the public Replit URL (to the homepage of your app) into the challenge screen to test it! Optionally, you may write your project on another platform, but it must be publicly visible for our testing.
```

Author: This does indeed feel more descriptive, I'll make the required changes.
